lib/matplotlib/backends/backend_agg.py

```diff
@@ -222,18 +222,13 @@ def draw_mathtext(self, gc, x, y, s, prop, angle):
 
     def draw_text(self, gc, x, y, s, prop, angle, ismath=False, mtext=None):
         # docstring inherited
-
         if ismath:
             return self.draw_mathtext(gc, x, y, s, prop, angle)
-
         flags = get_hinting_flag()
```

Comment:
I think you meant to remove this, since it's inlined below.

Reply:
oops, done.